Adyen/Components/Base/PrefilledShopperInformation.swift

```diff
@@ -0,0 +1,48 @@
+//
+// Copyright (c) 2021 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+import Foundation
+
+/// A structure that contains information about the shopper
+public struct PrefilledShopperInformation: ShopperInformation, BillingAddressInformation, SocialSecurityNumberInformation {
+
+    /// The name of the shopper.
+    public var shopperName: ShopperName?
+    
+    /// The email address.
+    public var emailAddress: String?
+    
+    /// The telephone number.
+    public var telephoneNumber: String?
+    
+    /// The billing address information.
+    public var billingAddress: AddressInfo?
```

Comment:
I am wandering if we should use `CNMutablePostalAddress` public here, mark `AddressInfo` as ":nodoc:" and use it only for JSON generation?

Reply:
`CNMutablePostalAddress` does not have all of the fields that `AddressInfo` has. no `houseNumberOrName`, `apartment`